ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/RpcMethod.java

```diff
@@ -43,8 +43,8 @@
   PRIV_DELETE_PRIVACY_GROUP("priv_deletePrivacyGroup"),
   PRIV_FIND_PRIVACY_GROUP("priv_findPrivacyGroup"),
   PRIV_DISTRIBUTE_RAW_TRANSACTION("priv_distributeRawTransaction"),
+  PRIV_GET_TRANSACTION_COUNT_LEGACY("priv_getTransactionCountLegacy"),
```

Comment:
Naming of this sounds a bit awkward. Perhaps priv_legacy_getTransactionCount or priv_legacyGetTransactionCount?